x-pack/plugins/cross_cluster_replication/public/app/sections/home/follower_indices_list/components/context_menu/context_menu.js

```diff
@@ -112,6 +112,7 @@ export class ContextMenuUi extends PureComponent {
                   <EuiContextMenuItem
                     icon="pause"
                     onClick={() => pauseFollowerIndex(activeFollowerIndices)}
+                    data-test-subj="ccrFollowerIndexContextMenuPauseActionButton"
```

Comment:
If we make that change, then we can dynamically prepend the supplied `data-test-subj` to make these values unique too. For example:

```jsx
data-test-subj={`${this.props['data-test-subj']}PauseActionButton`}
```